### PR TITLE
chore: update to latest changes from template

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
+We start from first principles and plan to avoid some pitfalls learned in maintaining that repo:
+
+- Use a toolchain consisting of off-the-shelf, pre-built layer and container manipulation tools (primarily [crane]).
+- Don't write language-specific rules, as we cannot be experts on all languages, nor can users deal with the versioning issues
+  that come with dependencies we might take on the rules for those languages.
+- Don't be docker-specific, now that it has a commercial license and other container runtimes exist.
+- Use our toolchain hermetically: don't assume there is a docker pre-installed on the machine.
+- Keep a tight complexity budget for the project so we are able to commit to effective maintenance.
+
+[crane]: https://github.com/google/go-containerregistry/blob/main/cmd/crane/doc/crane.md
```
Comment (thesayyn) [inline README.md:17]: I am not sure about crane yet. the combination of skopeo and umoci falls stronger than crane. Shall we remove this statement for now? 

Also see: https://github.com/aspect-build/rules_container/issues/9

File: internal_deps.bzl [hunk 2]
```diff
@@ -7,19 +7,9 @@ statement from these, that's a bug in our distribution.
...
-    maybe(
-        http_archive,
-        name = "rules_pkg",
-        urls = [
-            "https://mirror.bazel.build/github.com/bazelbuild/rules_pkg/releases/download/0.5.1/rules_pkg-0.5.1.tar.gz",
-            "https://github.com/bazelbuild/rules_pkg/releases/download/0.5.1/rules_pkg-0.5.1.tar.gz",
-        ],
-        sha256 = "a89e203d3cf264e564fcb96b6e06dd70bc0557356eb48400ce4b5d97c2c3720d",
-    )
-
```
Comment (thesayyn) [inline internal_deps.bzl:22]: Atm, we use rules_pkg to build tar for layers. Perhaps this should go into `rules_container_deps`?

**Review by thesayyn — CHANGES_REQUESTED** [review]

File: internal_deps.bzl (hunk 2 above)
Comment (alexeagle) [reply internal_deps.bzl]: yes, I see, it's a runtime dependency so yes I'll move it

File: README.md (hunk 1 above)
Comment (alexeagle) [reply README.md:17]: sure I'll remove the specific

**Review by thesayyn — APPROVED** [review]

**Pull request merged**